pkg/node/node.go

```diff
@@ -57,6 +59,7 @@ type Bee struct {
 	p2pCancel        context.CancelFunc
 	apiServer        *http.Server
 	debugAPIServer   *http.Server
+	multiResolver    resolver.Interface
```

Comment:
The Bee struct only keeps the multi resolver in order to close it on Shutdown. As with other closers this should be `resolverCloser io.Closer`. Also, it is just a resolver, multi is specific to the implementation, which is here not relevant.